ra-optimisation/search-tree-rao/src/main/java/com/farao_community/farao/search_tree_rao/Leaf.java

```diff
@@ -99,7 +99,13 @@ public String getMessage() {
         // apply Network Actions on initial network
         networkActions.forEach(na -> na.apply(network));
         // It creates a new CRAC variant
-        raoData = new RaoData(network, parentLeaf.getRaoData().getCrac(), parentLeaf.getRaoData().getOptimizedState(), parentLeaf.getRaoData().getPerimeter(), parentLeaf.getRaoData().getReferenceProgram(), parentLeaf.getRaoData().getGlskProvider());
+        raoData = new RaoData(network,
+            parentLeaf.getRaoData().getCrac(),
+            parentLeaf.getRaoData().getOptimizedState(),
+            parentLeaf.getRaoData().getPerimeter(),
+            parentLeaf.getRaoData().getReferenceProgram(),
+            parentLeaf.getRaoData().getGlskProvider(),
+            raoParameters.getLoopflowCountries());
```

Comment:
if we keep loopflowCountries in RaoData, it would be more readable to copy parent leaf's list like with the other fields

Reply:
good point!